pandas/tests/frame/methods/test_transpose.py

```diff
@@ -57,6 +58,7 @@ def test_transpose_object_to_tzaware_mixed_tz(self):
         df2 = DataFrame([dti, dti2])
         assert (df2.dtypes == object).all()
         res2 = df2.T
+        print("\n", res2.dtypes, [dti.dtype, dti2.dtype])
```

Comment:
remove prints

Reply:
removed the prints
these are the only two tests failing (but not on my local, was trying to figure why)